WordPress/twentytwentyfive
# Try: Remove images to reduce zip size.

**Description**


Partial to https://github.com/WordPress/twentytwentyfive/issues/401

- Remove `image-from-rawpixel-id-8803077-original.webp` and replace it with `image-from-rawpixel-id-8802835-jpeg-scaled.webp`
- Replace location.webp with a cropped version export from the [Figma comp](https://www.figma.com/design/dzGCSntVch4EQdVERTqyVK/Twenty-Twenty-Five?node-id=3177-1815&t=hF3HJeTuwkQyKLmx-4).
- Remove avatar and use other image instead. [See comment](https://github.com/WordPress/twentytwentyfive/issues/401#issuecomment-2402079779).

**Screenshots**



https://github.com/user-attachments/assets/0e6fd8ad-1229-4a3b-8eaa-3d1bd8dde85f


**Testing Instructions**

1. Create a page.
2. Add the following patterns: "Heading and paragraph with image on the right", "2 columns with avatar" and "Contact location and link".
3. Confirm that they're using the right images. That the images that are supposed to be removed are removed from the theme and the readme.
